.github/workflows/python_lint_test.yml

```diff
@@ -0,0 +1,38 @@
+# Workflow to run linting and tests on python code
+
+name: Python Workflow
```

Comment:
Maybe split this PR to two and commit this workflow separately first - so this workflow will run and test the second PR already.

Reply:
Oh, it actually ran for this PR already.